frontend-src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+import PlayBack from './Components/Playback/Playback';
+
+
+class App extends React.Component {
+  
```

Comment:
Just a small nitpick, there seem to be an excessive amount of newlines between some parts. If you have time and don't mind going through and deleting them, I'd appreciate it ^.^;

Reply:
I think I got most of them. :)